common/static/js/spec_helpers/ajax_helpers.js

```diff
@@ -109,6 +109,10 @@ define(['sinon', 'underscore'], function(sinon, _) {
         );
     };
 
+    respondWithNoContent = function(requests, requestIndex) {
+        respondToDelete(requests, requestIndex);
```

Comment:
This confused me because responding with no content has nothing to do with deletes. I think it would be clearer if you switched this around so that the default logic was in ``respondWithNoContent``, and then ``respondToDelete`` could just call ``respondWithNoContent``.